api/client/cli.go

```diff
@@ -125,6 +125,12 @@ func (cli *DockerCli) ImagesFormat() string {
 	return cli.configFile.ImagesFormat
 }
 
+// ServiceInspectFormat returns the format string specified in the configuration.
+// String contains columns and format specification, for example {{ID}}\t{{Name}}.
+func (cli *DockerCli) ServiceInspectFormat() string {
+	return cli.configFile.ServiceInspectFormat
+}
```

Comment:
@dnephin the PR is not merged either 👼 (waiting for the release to update and merge)